doit/task.py

```diff
@@ -407,8 +407,9 @@ def check_attr(task, attr, value, valid):
         @param valid (list): of valid types/value accepted
         @raises InvalidTask if invalid input
         """
-        if type(value) in valid[0]:
-            return
+        for t in valid[0]:
```

Comment:
for-loop not required. `isinstance()` second attribute can be a tuple of types.

Reply:
Didn't know! Changed it, thanks!